synapse/models/inet.py

```diff
@@ -2718,7 +2718,7 @@ def getModelDefs(self):
                         ('contacts', ('array', {'type': 'inet:whois:ipcontact', 'uniq': True, 'sorted': True}), {
                             'doc': 'Additional contacts from the record.',
                         }),
-                        ('country', ('str', {'lower': True, 'regex': '^[a-z]{2}$'}), {
+                        ('country', ('iso:3166:cc', {}), {
```

Comment:
This was modeled as an arbitrary string on purpose to be able to record potentially denormalized values present in whois records. That being said, this form could def benefit from adding the additional secondary properties we use for normalized / entity resolved location values :)  I'll discuss this with the team, thanks for the input!